packages/discussions/src/utils/platform.ts

```diff
@@ -27,6 +27,9 @@ export function reduceByGroupMembership(
  * @param {IUser} user
  * @return {*}  {boolean}
  */
+// NOTE: this is not the same as @esri/arcgis-rest-portal isOrgAdmin,
+// which first resolves `user` from `IUserRequestOptions` to make this determination
+// https://github.com/Esri/arcgis-rest-js/blob/7ab072184f89dcb35367518101ee4abeb5a9d112/packages/arcgis-rest-portal/src/sharing/helpers.ts#L45
```

Comment:
thanks also for this comment. I've opened an issue for this https://github.com/Esri/arcgis-rest-js/issues/818